frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseCheckoutAction.java

```diff
@@ -0,0 +1,131 @@
+package com.virtuslab.gitmachete.frontend.actions.base;
```

Comment:
Btw, pls also add an entry to CHANGE-NOTES.md and bump version.gradle.kts to 3.1.0 (as this constitutes a new user-facing feature, hence a MINOR bump is necessary)